precompiles/avs/IAVSManager.sol

```diff
@@ -7,12 +7,26 @@ address constant AVSMANAGER_PRECOMPILE_ADDRESS = 0x00000000000000000000000000000
 IAVSManager constant AVSMANAGER_CONTRACT = IAVSManager(
     AVSMANAGER_PRECOMPILE_ADDRESS
 );
-
 /// @author Exocore Team
 /// @title AVS-Manager Precompile Contract
 /// @dev The interface through which solidity contracts will interact with AVS-Manager
 /// @custom:address 0x0000000000000000000000000000000000000901
 interface IAVSManager {
+    event AVSRegistered(address indexed avsAddr, string sender, string avsName);
+    event AVSUpdated(address indexed avsAddr, string sender, string avsName);
+    event AVSDeregistered(address indexed avsAddr, string sender, string avsName);
+    event OperatorJoined(address indexed avsAddr,string sender);
+    event OperatorOuted(address indexed avsAddr,string sender);
+    event TaskCreated(string sender,uint64 taskId,address indexed taskContractAddress,string name,bytes hash,
+        uint64 taskResponsePeriod,uint64 taskChallengePeriod,uint64 thresholdPercentage,uint64 taskStatisticalPeriod
+    );
```

Comment:
It makes sense to make this edit, definitely points 2 and 3.

Regarding point 1, the `sender` is an `exo1....` address. However, within our chain, each address can be represented as a hex address **AND** a bech32 encoded address. In Solidity, the former type is well defined and can be used well, however, the latter is not easy to use (for example, even checking whether two strings are equal is not directly permitted in Solidity). It would make sense to use the hex version, if possible, within Solidity and the bech32 version elsewhere. Given that these are events which Solidity cannot access, I don't see any reasons to switch to hex here. Additionally, `indexed` keyword will hash the string leaving the original irrecoverable. What do you think @coderabbitai ?

Reply:
Given that these events are intended for external consumption and that Solidity does not interact with them directly, it makes sense to retain the sender as a string without the indexed keyword.

Reply:
What about points 2 and 3 @trestinlsd ? Do you agree or disagree with them ?

Reply:
Discussed that these are not super relevant and the number of members in the event does not surpass Solidity limits. Closing.